### PR TITLE
remove useless macros

### PR DESCRIPTION
**Pull request by bmorel**
Macros only used once just makes the code harder to read (and patch)

### CODE REVIEW

**Comment by MoonPadUSer** [comment]
IMO this seems reasonable to do, but please wait for @TheAssassin to confirm

File: src/shared/glemu.h [hunk 1]
```diff
@@ -40,23 +40,19 @@ namespace gle
...
-        static inline void name##suffix(type x) { glVertexAttrib1##suffix##_(index, x); } \
-        static inline void name##suffix(type x, type y) { glVertexAttrib2##suffix##_(index, x, y); } \
-        static inline void name##suffix(type x, type y, type z) { glVertexAttrib3##suffix##_(index, x, y, z); } \
-        static inline void name##suffix(type x, type y, type z, type w) { glVertexAttrib4##suffix##_(index, x, y, z, w); }
     #define GLE_INITATTRIBF(name, index) \
-        GLE_INITATTRIB(name, index, f, float) \
+        static inline void name##f(float x) { glVertexAttrib1##f##_(index, x); } \
+        static inline void name##f(float x, float y) { glVertexAttrib2##f##_(index, x, y); } \
+        static inline void name##f(float x, float y, float z) { glVertexAttrib3##f##_(index, x, y, z); } \
+        static inline void name##f(float x, float y, float z, float w) { glVertexAttrib4##f##_(index, x, y, z, w); } \
```
Comment (robalni) [inline src/shared/glemu.h:47]: `f` is not a macro parameter anymore so it doesn't need to be glued using `##`.

```suggestion
        static inline void name##f(float x, float y, float z, float w) { glVertexAttrib4f_(index, x, y, z, w); } \
```

Comment (bmorel) [reply src/shared/glemu.h]: Nice catch, thanks!

File: src/shared/glemu.h [hunk 2]
```diff
@@ -44,7 +44,7 @@ namespace gle
         static inline void name##f(float x) { glVertexAttrib1##f##_(index, x); } \
         static inline void name##f(float x, float y) { glVertexAttrib2##f##_(index, x, y); } \
         static inline void name##f(float x, float y, float z) { glVertexAttrib3##f##_(index, x, y, z); } \
-        static inline void name##f(float x, float y, float z, float w) { glVertexAttrib4##f##_(index, x, y, z, w); } \
+        static inline void name##f(float x, float y, float z, float w) { glVertexAttrib4f_(index, x, y, z, w); } \
```
Comment (robalni) [inline src/shared/glemu.h:47]: It was not only this line; the ones above have the same thing.

Comment (bmorel) [reply src/shared/glemu.h:47]: well, turns out that most of that code is *plain useless* so I'm spending some extra time remove unneeded stuff.

**Comment by bmorel** [comment]
Closing because I don't know how to change source branch

**Pull request closed without merging**